bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wh_M125_ToAA_Tobbgg_LO_MLM/Wh_M125_ToAA__AMASS__Tobbgg_run_card.dat

```diff
@@ -0,0 +1,269 @@
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .false.     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+      500000 = nevents ! Number of unweighted events requested 
+      0       = iseed   ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+        1     = lpp1    ! beam 1 type 
+        1     = lpp2    ! beam 2 type
+     6500     = ebeam1  ! beam 1 total energy in GeV
+     6500     = ebeam2  ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+  lhapdf = pdlabel ! PDF set
+ $DEFAULT_PDF_SETS = lhaid
+ $DEFAULT_PDF_MEMBERS  = reweight_PDF
+#*********************************************************************
+# Renormalization and factorization scales                           *
+#*********************************************************************
+ F        = fixed_ren_scale  ! if .true. use fixed ren scale
+ F        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.1880  = scale            ! fixed ren scale
+ 91.1880  = dsqrt_q2fact1    ! fixed fact scale for pdf1
+ 91.1880  = dsqrt_q2fact2    ! fixed fact scale for pdf2
+ 1        = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 1        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
```

Comment:
why is ickkw enabled? do you planned to use mlm?

Reply:
We want to use MLM matching, so we have done ickkw = 1

Reply:
Do you have other proc_cards with multiple jets to show that this uses MLM?

Reply:
All other proc_cards with multiple jets use ickkw = 0, but I don't know, is it a problem if we want to use MLM?

Reply:
I am not fully understanding what you mean.
If a process includes additional partons in ME level (e.g. p p > X, p p > X j, p p X j j) -> this needs MLM jet merging.
If it's not a such process, (e.g. p p > X only) -> this doesn't need MLM jet merging

Reply:
Your process is "generate p p  > V h" (decay of V or higgs is irrelevant here).

And you do have additional partons like "p p  > V h j". So you don't need to + should not use MLM jet merging

Reply:
Ok, thanks, I have done ickkw = 0 & modified the run card accordingly.